client/src/includes/tabs.js

```diff
@@ -0,0 +1,339 @@
+/** Example Markup:
+ <div class="w-tabs" data-tabs data-tabs-animate>
+ <div class="w-tabs__wrapper w-overflow-x-auto u-scrollbar-thin">
+ <div role="tablist" class="w-tabs__list">
+ {% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with tab_id='slug' title='Tab Title' %}
+ </div>
+ <!-- Tab Content -->
+ <section id="tab-{{ tab_id }}" class="w-tabs__panel" role="tabpanel" aria-labelledby="tab-label-{{ tab_id }}" hidden>
+ {{ content }}
+ </section>
+ </div>
+ </div>
+ */
```

Comment:
This already looks out of date (`w-tabs__wrapper w-overflow-x-auto u-scrollbar-thin`), so we should remove it and have a comment that explains how the tabs are built instead, based on https://w3c.github.io/aria-practices/#tabpanel.

Reply:
I made an attempt to revise this but I wasn't quite sure what you meant :/